setup.py

```diff
@@ -28,7 +28,7 @@
         'django-compressor',
         'django-templatetag-handlebars',
         'django_mobileesp',
-        'django-userservice',
+        'django-userservice==1.2.1',
```

Comment:
Since this is a supporting app, you might want to change this to 'django-userservice>=1.2.1,<2.0'